src/main/java/org/javarosa/form/api/FormEntryController.java

```diff
@@ -106,7 +106,7 @@ public int answerQuestion(FormIndex index, IAnswerData data) {
                     model.getForm().copyItemsetAnswer(q, element, data);
                 } catch (InvalidReferenceException ire) {
                     ire.printStackTrace();
-                    throw new RuntimeException("Invalid reference while copying itemset answer: " + ire.getMessage());
+                    throw new RuntimeException("Invalid reference while copying itemset answer: " + ire.getMessage() + " Reference: " + ire.getInvalidReference().toString());
```

Comment:
I'm not clear on whether the reference here can be null, which should likely be protected so we don't obfuscate the runtime expresson.

I think "toString()" is an implicit, here so it may not be necessary anyway.